### PR TITLE
[AUD-1581] Add social proof UI

### PR DESCRIPTION
**Pull request by raymondjacobson**
### Description

Add modal for social proof

### Dragons

*Is there anything the reviewer should be on the lookout for? Are there any dangerous changes?*

### How Has This Been Tested?

*Please describe the tests that you ran to verify your changes. Provide repro instructions & any configuration.*

Locally as much as possible. Need staging env for the oauth integrations to work properly.

### How will this change be monitored?

*For features that are critical or could fail silently please describe the monitoring/alerting being added.*

### CODE REVIEW

File: packages/web/src/pages/audio-rewards-page/Tiles.tsx [hunk 1]
```diff
@@ -144,7 +144,7 @@ export const WalletTile = ({ className }: { className?: string }) => {
               [styles.balanceDisabled]: !hasBalance
             })}
             text={messages.sendLabel}
-            isDisabled={!hasBalance}
+            // isDisabled={!hasBalance}
```
Comment (raymondjacobson) [inline packages/web/src/pages/audio-rewards-page/Tiles.tsx:147]: enable before merge

File: packages/web/src/pages/audio-rewards-page/components/SendInputBody.tsx [hunk 2]
```diff
@@ -152,7 +152,7 @@ const validateSendAmount = (
   )
   if (!sendWeiBN) return 'MALFORMED'
   if (minWeiBN && sendWeiBN.lt(minWeiBN)) return 'LESS_THAN_MIN'
-  if (sendWeiBN.gt(balanceWei)) return 'INSUFFICIENT_BALANCE'
+  // if (sendWeiBN.gt(balanceWei)) return 'INSUFFICIENT_BALANCE'
```
Comment (raymondjacobson) [inline packages/web/src/pages/audio-rewards-page/components/SendInputBody.tsx:155]: enable before merge

File: packages/web/src/components/social-proof/SocialProof.tsx [hunk 3]
```diff
@@ -0,0 +1,227 @@
...
+  )
+  const onTwitterLogin = useCallback(
+    (uuid: string, profile: TwitterProfile) => {
+      dispatch(twitterLoginAction({ uuid, profile }))
+    },
+    [dispatch]
+  )
+
+  const [error, setError] = useState('')
+  const [status, setStatus] = useState('')
```
Comment (dylanjeffers) [inline packages/web/src/components/social-proof/SocialProof.tsx:134]: lets maybe get a neutral status, so we are not checking status === '' ?

File: packages/web/src/components/social-proof/SocialProof.tsx [hunk 4]
```diff
@@ -0,0 +1,227 @@
...
+        error: error.message
+      })
+      record(trackEvent)
+    },
+    [record, setError, setStatus, handle]
+  )
+
+  const instagramLogin = useCallback(
+    (uuid: string, profile: InstagramProfile) => {
+      if (!handle) return
```
Comment (dylanjeffers) [inline packages/web/src/components/social-proof/SocialProof.tsx:159]: not sure of full context here, but do we want this to trigger failure notice if we dont have handle?

File: packages/web/src/components/social-proof/SocialProof.tsx [hunk 5]
```diff
@@ -0,0 +1,227 @@
...
+  )
+
+  useEffect(() => {
+    if (status === Status.SUCCESS) {
+      onSuccess()
+    }
+  }, [onSuccess, status])
+
+  let body = null
+  if (status === Status.LOADING || handle === null) {
```
Comment (dylanjeffers) [inline packages/web/src/components/social-proof/SocialProof.tsx:198]: so if the handle is missing that means the selector has triggered i fetch for it? just checking that we dont get infinite loading or something

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/rj-aud-1581

File: packages/web/src/store/token-dashboard/sagas.ts [hunk 6]
```diff
@@ -97,14 +98,22 @@ function* send() {
   })
 
   if (error) {
-    const errorState: ModalState = {
-      stage: 'SEND',
-      flowState: {
-        stage: 'ERROR',
-        error: error.payload.error ?? ''
+    if (error.payload.error === 'Missing social proof') {
+      yield all([
```
Comment (dylanjeffers) [inline packages/web/src/store/token-dashboard/sagas.ts:102]: yield all for first two and not third? checking if im missing some context there, just ensures they finish first?

**Review by dylanjeffers — APPROVED** [review]
nice work!

**Pull request merged**